### DISCUSSION TITLE
Consider switching to better maintained alternatives to normalize.css

### DISCUSSION THREAD

**Discussion by mcmxcdev** [mantinedev/mantine#1598] [Feature requests]
https://github.com/necolas/normalize.css hasn't had a new release since 2018. 
There are open issues about the lacking project maintenance: https://github.com/necolas/normalize.css/issues/880

It would be great to investigate alternatives, e.g. https://github.com/sindresorhus/modern-normalize

**Comment by rtivital** [comment]
Mantine does not use nomalize.css, we have our own CSS reset that is similar to normalize, but it is not identical. I haven't experienced any issues with it, so don't really understand why we need to update any styles if nothing is broken.

**Reply by mcmxcdev** [reply]
https://mantine.dev/theming/mantine-provider/#normalizecss-and-global-styles makes it sound like Mantine does.

If that's not the case, all good!
